### PR TITLE
bio updates

### PR DESCRIPTION
**Pull request by KafkaNoNeko**
Updated all the necessary information for everyone

### CODE REVIEW

File: content/bio.md [hunk 1]
```diff
@@ -1,70 +1,81 @@
...
+| Parsa Rajabi | Parsa-Rajabi |ParsaRajabiPR@gmail.com |
 | Paula Wong-Chung | KafkaNoNeko | paula.wong-chung@alumni.ubc.ca |
-| Edris Wu | edriswu | |
+| Edris Wu | edriswu | edriswu@gmail.com |
 
 ## Bios
+****
 
-### Principal Investigator
+## Principal Investigator
```
Comment (firasm) [inline content/bio.md]: This should be ###

File: content/bio.md [hunk 2]
```diff
@@ -1,70 +1,81 @@
...
-<img align="left" width="150" height="150" style="padding-right: 20px" src="../images/placeholder.png"> </img>
+<img align="left" width="150" height="150" style="padding-right: 20px" src="../images/dr.moosvi.png"> </img>
 
 #### **Dr. Firas Moosvi**
 
-Dr. Moosvi's bio.<br><br><br><br><br><br><br>
+Firas Moosvi is a Lecturer at UBC Okanagan in the Computer Science, Math, Physics, and Statistics department. He teaches courses in Physics, Data Science, and Computer Science at the undergraduate and graduate levels. He has many research interests and they fall under three main umbrellas: medical imaging for applications in cancer, scholarship of teaching and learning (SoTL), and learning analytics. Most recently, he is looking at how the field of learning analytics can provide insight to surface and reduce inequities in STEM programs. Firas has a deep appreciation for data visualization, active learning, and open source projects.<br><br><br><br><br><br><br>
 
-### Graduate students
+## Graduate students
```
Comment (firasm) [inline content/bio.md]: This should be ###

File: content/bio.md [hunk 3]
```diff
@@ -1,70 +1,81 @@
...
 
-<img align="left" width="150" height="150" style="padding-right: 20px" src="../images/placeholder.png"> </img>
+<img align="left" width="150" height="150" style="padding-right: 20px" src="../images/michael.png"> </img>
 
 #### **Michael Kudla**
 
 I am a PhD student in MedPhys. I am developing a patient specific technology suite for low cost imaging and surgical tool manufacturing (3D printed) for treatment of gynecological malignancies. I am into rock climbing and ski-touring. <br><br><br><br><br>
 
-### Undergraduate students
+## Undergraduate students
```
Comment (firasm) [inline content/bio.md]: This should be ###

**Review by firasm — CHANGES_REQUESTED** [review]
Thanks @KafkaNoNeko - could you change the header levels so that it doesn't skip from ## to #### ?

**Comment by KafkaNoNeko** [comment]
@firasm done :)

**Review by firasm — APPROVED** [review]

**Pull request merged**